j5/components/piezo.py

```diff
@@ -61,4 +61,14 @@ def interface_class() -> Type[PiezoInterface]:
     def buzz(self, board: Board, identifier: int,
              duration: timedelta, pitch: Pitch) -> None:
         """Queue a note to be played."""
-        self._backend.buzz(board, identifier, duration, pitch)
+        if type(pitch) is int:
+            frequency = pitch
+        elif type(pitch) is Note:
+            frequency = pitch.value  # type: ignore
```

Comment:
I worked with @HU90m to try and get these types to work with no success. If we can't fix them before merging, I suggest opening an issue so we don't forget about them.

@sedders123 can you help here?

Reply:
Unfortunately `enum` values are a [bit of a pain](https://github.com/python/mypy/issues/3546) in mypy, though there are [plans](https://github.com/python/mypy/issues/3541) to try and improve it with plugins. In the meantime would a `cast` to type `int` work here?

Reply:
We tried `cast` earlier and unfortunately it didn't work.

Reply:
Aha. We should try `IntEnum` as it has the type of `value` defined as `int` in https://github.com/python/typeshed/blob/master/stdlib/3/enum.pyi#L35

Reply:
```python
def buzz(self, board: Board, identifier: int,
             duration: timedelta, pitch: Pitch) -> None:
        """Queue a note to be played."""
        if type(pitch) is int:
            frequency = cast(int, pitch)
        elif type(pitch) is Note:
            frequency = cast(Enum, pitch).value
        else:
            raise TypeError

        if frequency < 0:
            raise ValueError
        else:
            self._backend.buzz(board, identifier, duration, pitch)
```

The above code seems to pass the linter. Had to cast on both `int` case and `Note` case.

Reply:
Interesting. I couldn't get the above to work.

My solution is PR to this branch #121.

Reply:
👍 Like the look of #121